src/app/components/ErrorPage/index.jsx

```diff
@@ -45,21 +49,21 @@ const ErrorPage = ({
           <StatusCode script={script}>{statusCode}</StatusCode>
           {title}
         </ShortHeadline>
-        <Paragraph script={script}>{message}</Paragraph>
+        <StyledParagraph script={script}>{message}</StyledParagraph>
         <ul>
           {solutions.map(text => (
-            <Paragraph script={script} as="li" key={idSanitiser(text)}>
+            <StyledParagraph script={script} as="li" key={idSanitiser(text)}>
```

Comment:
Psammead paragraph has a required prop for `service` now.

I havn't tested it locally but would passing the current service here fix this, rather than creating a new styledParagraph ?

Reply:
Thanks. I've made the change.